docs/home/contribute/write-new-topic.md

```diff
@@ -22,17 +22,17 @@ is the best fit for your content:
 
   <tr>
     <td>Task</td>
-    <td>A task page shows how to do a single thing, typically by giving a short sequence of steps. Task pages have minimal explanation, but often provide links to conceptual topics that provide related background and knowledge.</td>
+    <td>A task page shows how to do a single thing. The idea is to give readers a sequence of steps that they can actually do as they read the page. A task page can be short or long, provided it stays focused on one area. In a task page, it is OK to blend brief explanations with the steps to be performed, but if you need to provide a lengthy explanation, you should do that in a concept topic. Related task and concept topics should link to each other. For an example of a short task page, see <a href="/docs/tasks/configure-pod-container/configure-volume-storage/">Configure a Pod to Use a Volume for Storage</>. For an example of a longer task page, see <a href=/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/">Configure Liveness and Readiness Probes</a></td>
```

Comment:
The *Configure a Pod to Use a Volume for Storage* link is broken. I think it's because the closing tag should be `</a>`.